scripts/validate_docs.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
-
+# TODO: Make this print out if commands are missing
```

Comment:
What do you mean by commands are missing?

Reply:
Ah, to print a nicer error if `npm` or `git` are missing